mod/allfriends.php

```diff
@@ -26,7 +28,8 @@ function allfriends_content(&$a) {
 		intval(local_user())
 	);
 
-	if(! count($c))
+	if (! count($c)) {
+	}
```

Comment:
The closing bracket should be moved after the following `return;` statement. Thanks for adding them though!